### ISSUE TITLE
Feature Request: Ability to optionally display "Description" in addition to the "Message" attribute

### ISSUE THREAD

**Issue by ahd71** [atomic7777/atomic_calendar#14]

**Comment by atomic7777** [comment]
I don't understand, could you write a bit more about what do you expect?

**Comment by ahd71** [comment]
Oh, sorry. It seems i didn't provide the description - just the subject by mistake. Let me elaborate:

What I'm asking for is a posibility in your calendar card to show also the "description" attribute in addition to "message", "start_time", "end_time" and "location". 

I suggest that it is optional to display it (possible to toggle/format just like some of the other attributes)

One calendar entity contains for example the following information for a event using the "description" attribute which in some cases contains valuable information:

`message: Träning - BP Ungdom
all_day: false
offset_reached: false
start_time: 2019-06-18 19:00:00
end_time: 2019-06-18 21:00:00
location: Hässelängen BP
description: Samlingstid
2019-06-18 18:50
Övrig info
OBS!!  Specialupplägg se mailet från Lotta!!  
friendly_name: Fotboll`

thank you /Anders

**Comment by leloberg** [comment]
I second this request.

**Comment by atomic7777** [comment]
I started to work on this. But, where it should be?
like this?
![desc](https://user-images.githubusercontent.com/11677097/60717490-377aa600-9f22-11e9-89ee-247edea91393.png)

**Comment by leloberg** [comment]
Think you need to experiment with the position. Maybe try that position first, and we'll see how it looks? The other alternative I guess is right under the title.

**Comment by ahd71** [comment]
I vote for last, like you suggested with the arrow, as the text may be long. 
Perhaps also styled so it is less dominant than then the title and time. Slightly lighter grey then the title for example or font size 20% smaller?

**Comment by atomic7777** [comment]
Try it now. I added "showDescription" optional setting, and "descSize", "descColor", it's 80% of standard text size.
It should look like this:
![desc](https://user-images.githubusercontent.com/11677097/60862843-2df18680-a21f-11e9-9048-ceb67dfd5427.png)

**Comment by leloberg** [comment]
For some reason I can't get this working. I've tried to restart HA too, and remove all calendars and re-add them.

**Comment by ahd71** [comment]
It works for me! :-)

Steps to install:

 - download the file (in the root folder, not in the dist folder!) - https://raw.githubusercontent.com/atomic7777/atomic_calendar/master/atomic_calendar.js
 - save in your lovelace directory - pay attention to _ vs - in the filename which seems to be different from the DIST directory
 - update the RAW lovelace configuration to load the component (I appended a version number to make it unique so that HA reloads the file, otherwise it is old file cached)
    url: /local/lovelace/atomic-calendar.js?v=0.8.4
- add "showDescription: true" in the atomic-calendar lovelace card configuration

Thank you very much for the enhancement @atomic7777 !

**Comment by leloberg** [comment]
Yes, that would work for me too, but I want to use HACS. I'll just give it some time to catch up, might be some delay for all I know. 😉

**Comment by atomic7777** [comment]
Sorry I made a new release without changing the file in /dist folder. Now it should be ok event with HACK (I thought HACS is downloading the file from release, but it's not)

**Comment by leloberg** [comment]
Perfect! It works :) Looks nice too, great job!
